doc/developer/topotests.rst

```diff
@@ -39,8 +37,7 @@ Installing Topotest Requirements
    python3 -m pip install 'pytest-xdist>=2.3.0'
    python3 -m pip install 'scapy>=2.4.5'
    python3 -m pip install xmltodict
-   # Use python2 pip to install older ExaBGP
-   python2 -m pip install 'exabgp<4.0.0'
+   python3 -m pip install 'exabg>=4.2.11'
```

Comment:
I would suggest to install exabgp 4.2.11 from our OSR repository which has the required patches:
`pip3 install git+https://github.com/opensourcerouting/exabgp@4.2.11-frr`